src/auro-menu.js

```diff
@@ -242,6 +242,31 @@ class AuroMenu extends LitElement {
     });
   }
 
+  /**
+   * Used to make a menuoption selection directly using a defined `value`.
+   * @param {String} value - String that must match the value attribute on
+   * a single menuoption.
```

Comment:
```suggestion
```

Reply:
This is intentional because the selection process is unreliable if more than one menu option have the same value (which we have no control over).

Reply:
This line was deleted because it wasn't needed with the suggested rewording.

Reply:
It is still needed - though I understand it would be unusual for menuoptions to have duplicate values, there is no enforcement that they are unique. The extra bit of text is there because if you don't match the value of `a single menuoption` then the selectbyvalue method may produce unexpected results.